### ISSUE TITLE
Prisma 1.34.7 breaks services with a service secret (token invalid)

### ISSUE THREAD

**Issue by gerwim** [prisma/prisma#4860]
**Describe the bug**
Prisma `1.34.7` breaks services with a service secret.

**To Reproduce**
1. Deploy Prisma `1.34.7` (sample compose file):
```
version: '3'
services:
  prisma:
    image: prismagraphql/prisma:1.34.7
    restart: always
    ports:
      - "4466:4466"
    environment:
      PRISMA_CONFIG: |
        port: 4466
        databases:
          default:
            connector: mysql
            host: mysql
            port: 3306
            user: root
            password: prisma
            migrations: true
  mysql:
    image: mysql:5.7
    restart: always
    ports:
      - "3306:3306"
    environment:
      MYSQL_ROOT_PASSWORD: prisma
    volumes:
      - data-mysql:/var/lib/mysql

volumes:
  data-mysql:
    driver: local
```
2. Deploy a service with a `secret`, sample `prisma.yml`:
```
endpoint: http://localhost:4466
secret: devSecret
datamodel:
  - datamodel.prisma
```
3. Run `prisma token`
4. Explore the Prisma admin interface with this token or perform queries in the playground, which will return `Your token is invalid. It might have expired or you might be using a token from a different project.`

Reverting back to < `1.34.7` (e.g. `1.34.6`) work fine.

**Expected behavior**
The query / actions work.

**Versions (please complete the following information):**
 - Connector: both `Postgres` and `MySQL`
 - Prisma Server: [e.g. `1.34.7`]
 - `prisma` CLI: ` prisma/1.34.7 (windows-x64) node-v10.15.3`
 - OS: `Windows 10`

**Comment by chanlito** [comment]
Facing same issue, I'm downgrading to a prev version.

**Comment by pavelpashkovsky** [comment]
Experiencing the same issue

**Comment by kaienbcjdoapwkfldue** [comment]
Same here. This one screwed me up really hard yesterday. Lesson learned the hard way: update dependencies atomically and **check!** afterwards.

**Comment by jimhill** [comment]
Yep we have just spent two days trying to get our head around this and only able to reproduce today.

**Comment by rares-lupascu** [comment]
same here ... just fyi ... is there an eta on this?

**Comment by pantharshit00** [comment]
We are looking into this. We will release a patch as soon as this is fixed.

**Comment by pantharshit00** [comment]
Fixed in `1.34.8`
